models/demos/t3000/falcon40b/tests/test_falcon_causallm.py

```diff
@@ -351,7 +341,7 @@ def test_FalconCausalLM_inference(
 
     input_shape = [batch, seq_len]
     model_config = get_model_config(model_config_str, llm_mode, input_shape, num_devices)
-    devices = get_devices_for_t3000(all_devices, num_devices)
+    devices = t3k_device_mesh.get_devices()
```

Comment:
We should remove `num_devices` as argument and use the number of devices in the `t3k_device_mesh` to pass into `get_model_config` instead.
Same for every other test.

Reply:
I agree but I would put a separate PR for that.